h/accounts/models.py

```diff
@@ -123,6 +123,17 @@ def subscriptions(self, value):
         else:
             self.status = (self.status or 0) & ~0b100
 
+    @property
+    def invited(self):
+        return bool((self.status or 0) & 0b100)
```

Comment:
God this is ugly. Also, even if we must continue using this godawful bitfield, this should clearly be `0b1000`, not `0b100`.